### ISSUE TITLE
test_lowest_common_hypernyms fails

### ISSUE THREAD

**Issue by kmike** [nltk/nltk#445]
test_lowest_common_hypernyms introduced in https://github.com/nltk/nltk/commit/609fe05dfcf32c0b11a7de82124f02cf86f0434b fails for me (both under Python 2.7 and 3.3):

```
> tox -e py27-nodeps -- nltk.test.unit.test_wordnet
GLOB sdist-make: /Users/kmike/svn/nltk/setup.py
py27-nodeps inst-nodeps: /Users/kmike/svn/nltk/.tox/dist/nltk-3.0a2.zip
py27-nodeps runtests: commands[0]
F
======================================================================
FAIL: This tests wordnet's lowest_common_hypernyms method. It should
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/kmike/svn/nltk/.tox/py27-nodeps/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/kmike/svn/nltk/nltk/test/unit/test_wordnet.py", line 37, in test_lowest_common_hypernyms
    assert len(lch) == 1
AssertionError
```

I also think that this test could work better written as doctest (preferable) or as many small unit tests. In both cases we'll be able to see individual "small" failings - now the test stops working on first error. There is no need to put this test to wordnet.doctest (it is slow to run as whole); I think a separate file should be ok if we decide to rewrite it as a doctest.

**Comment by dougalg** [comment]
I'd be happy to do this over the next couple of days if you'd like. Sorry for introducing the error, I was sure I'd run the test a final time before submitting, but I must have done something wrong.

**Comment by stevenbird** [comment]
@kmike is this resolved?

**Comment by kmike** [comment]
Yes, this is fixed by #465.
